.github/workflows/publish-to-npm-on-tag.yml

```diff
@@ -76,6 +78,8 @@ jobs:
 
       - name: Build
         run: npm run build
+        env:
+          NODE_ENV: 'production'
```

Comment:
why is it needed? would it cause minification?

Reply:
I see in that case it is fine but AFAIK we have no source map support at all in the project right now